packages/client/package.json

```diff
@@ -35,8 +35,8 @@
     "@osdk/api": "workspace:~",
     "@osdk/client.unstable": "workspace:*",
     "@osdk/generator-converters": "workspace:*",
-    "@osdk/internal.foundry.core": "2.5.0",
-    "@osdk/internal.foundry.ontologiesv2": "2.5.0",
+    "@osdk/internal.foundry.core": "2.6.0-beta.0",
+    "@osdk/internal.foundry.ontologiesv2": "2.6.0-beta.0",
```

Comment:
We're fine with this on a beta release? I don't see why it matters so just checking

Reply:
That should be fine cuz these are beta clients anyways

Reply:
Also going to upgrade soon